4_arm.ipynb

```diff
@@ -357,6 +644,19 @@
     "逆運動学の解が得られなかった場合（目標の手先座標の設定が適切でなかった場合）は、長さゼロの配列が返ってきますので、目標を修正して再計算してください。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "After setting the target, we will start the inverse kinematics calculation.\n",
+    "\n",
+    "If the calculation is successful, the calculated trajectory will be returned. Variable `plan` stores the information on the target values of all joint angles for each time (called \"time trajectory\").\n",
+    "\n",
+    "If the inverse kinematics solution is not obtained (if the target hand coordinates are not set properly), an array of zero length will be returned. Please correct the target position and recalculate in such case:"
```

Comment:
"If the inverse kinematics solution is not obtained (if the target hand coordinates are not set properly), an array of zero length will be returned. Please correct the target position and recalculate in such a case:"